cfg/two_layers_gray.cfg

```diff
@@ -0,0 +1,19 @@
+#======== Layers ===============
+MaxLayers                     : 2
+MaxSublayers                  : 7
+AllIndependentLayersFlag      : 1
+#======== Alpha ================
```

Comment:
When adding an alpha channel layer, you would always want to send the SDI and ACI SEI messages, I don't think this should be configurable by the user. A better configuration interface would be the ability to define what layer is the alpha channel one:
#======== Layer-0 ===============
LayerId0                      : 0
#======== Layer-1 ===============
LayerId1                      : 1
#======== Alpha layer ================
LayerAlpha                 : 1


AlphaLayedId0